### PR TITLE
helm: Add subcomponent labels for celery beat and worker at deployment

### PR DESCRIPTION
**Pull request by al-cheb**
In scope of this https://github.com/DefectDojo/django-DefectDojo/pull/9078 PR only the deployment label was added. To use pod affinity/antiaffinity for celery components a label should be added at  `.spec.template.metadata.labels` level.

```
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    defectdojo.org/component: celery
    defectdojo.org/subcomponent: worker
spec:
  selector:
    matchLabels:
      defectdojo.org/component: celery
      defectdojo.org/subcomponent: worker
  template:
    metadata:
      labels:
        defectdojo.org/component: celery
        defectdojo.org/subcomponent: worker
```

1, The first label is for deployment itself, it gives label for that particular deployment. 
2. The second label is selector: matchLabels which tells the resources(service etc) to match the pod according to label.
3. The third label is podTemplate labels, the template is actually podTemplate. It describe the pod that it is launched.

### CODE REVIEW

**Comment by mtesauro** [comment]
Closing and re-opening to re-kick the tests that didn't seem to run correctly.

**Comment by al-cheb** [comment]
@mtesauro, Thank you.

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

File: helm/defectdojo/Chart.yaml
Comment (Maffooch) [inline helm/defectdojo/Chart.yaml:1]: Please do not change the chart version here

File: helm/defectdojo/Chart.lock
Comment (Maffooch) [inline helm/defectdojo/Chart.lock:1]: please revert this change as well as we do not want to rev versions here

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: helm/defectdojo/Chart.yaml
Comment (al-cheb) [reply helm/defectdojo/Chart.yaml:1]: Thanks, updated.

File: helm/defectdojo/Chart.lock
Comment (al-cheb) [reply helm/defectdojo/Chart.lock:1]: Thanks, updated.

**Review by Maffooch — APPROVED** [review]

**Comment by al-cheb** [comment]
@Maffooch, could you merge,please?

**Comment by mtesauro** [comment]
@al-cheb PRs require 4 approvals from the reviewers before being merged.

The only exception is library/module updates like dependabot and renovate.

**Comment by al-cheb** [comment]
@dsever, hey, could you please review?

**Review by hblankenship — APPROVED** [review]

**Pull request merged**
